### DISCUSSION TITLE
Allow partial email addresses for coupons

### DISCUSSION THREAD

**Discussion by dgkeyes** [duncanmcclean/simple-commerce#920] [Feature Requests]
When I create coupons, rather than have to manually connect customers, I sometimes want to make the coupons work for everyone at a certain organization. It would be nice if I could set them to work, say, for everyone with an rfortherestofus.com email.

**Comment by duncanmcclean** [comment]
This is now possible in the latest version! See #960. 🚀

**Reply by dgkeyes** [reply]
Awesome, thanks!
